extension/src/paymentHandler/cancel-payment.handler.js

```diff
@@ -44,6 +44,10 @@ function _createTransactionActions(paymentObject, pspReference) {
   return [
     pU.createChangeTransactionStateAction(transactionId, 'Pending'),
     pU.createChangeTransactionInteractionId(transactionId, pspReference),
+    pU.createChangeTransactionTimestampAction(
+      transactionId,
+      paymentObject.createdAt
```

Comment:
Since at this point we had already response from Adyen why we take timestamp from the payment object and not from the response or Adyen notification? Also, payment createdAt might be very different from the actual processing of the transaction, right?

Reply:
Actually at this point we don't have the timestamp from notification. We will have a transaction with Pending state. I would put the timestamp there like Hin did here so that we know when the transaction was added. It could be that the transaction will stay in Pending for some time.

Reply:
As discuss, we don't need to set timestamp in extension module explicitly. We just need the event date returned from notification module.
https://github.com/commercetools/commercetools-adyen-integration/pull/826/commits/9e35ab7f3f1285abb7119ef3a73d820293b3fba2